### PR TITLE
Issue829 sunriseset

### PR DESCRIPTION
**Pull request by kuzha**
Please have a look

### CODE REVIEW

File: Buildings/Controls/OBC/CDL/Utilities/SunRiseSet.mo [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  Modelica.SIunits.Angle zenAng "Zenith angle";
+  Modelica.SIunits.Angle altAng "Altitude angle";
+
+  Modelica.Blocks.Interfaces.RealInput nDay(quantity="Time", unit="s")
+    "Day number with units of seconds"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+  Modelica.Blocks.Interfaces.RealOutput sunRise(
+    final quantity="Time",
+    final unit="h",
```
Comment (mwetter) [inline Buildings/Controls/OBC/CDL/Utilities/SunRiseSet.mo]: This needs to be seconds

File: Buildings/Controls/OBC/CDL/Utilities/SunRiseSet.mo [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+  Modelica.SIunits.Time eqnTim "Equation of time";
+  Modelica.SIunits.Time locTim "Local time";
+  Modelica.SIunits.Time solTim "Solar time";
+  Modelica.SIunits.Angle houAng "Solar hour angle";
+  Modelica.SIunits.Angle decAng "Declination angle";
+  Modelica.SIunits.Angle zenAng "Zenith angle";
+  Modelica.SIunits.Angle altAng "Altitude angle";
+
+  Modelica.Blocks.Interfaces.RealInput nDay(quantity="Time", unit="s")
```
Comment (mwetter) [inline Buildings/Controls/OBC/CDL/Utilities/SunRiseSet.mo]: use "final unit..."

File: Buildings/Controls/OBC/CDL/Utilities/SunRiseSet.mo [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+  Modelica.Blocks.Interfaces.RealOutput sunRise(
+    final quantity="Time",
+    final unit="h",
+    displayUnit="h") "Sunrise time"
+    annotation (Placement(transformation(extent={{100,30},{120,50}})));
+  Modelica.Blocks.Interfaces.RealOutput sunSet(
+    final quantity="Time",
+    final unit="h",
```
Comment (mwetter) [inline Buildings/Controls/OBC/CDL/Utilities/SunRiseSet.mo]: Use `s` not `h`

**Comment by kuzha** [comment]
@mwetter Please have a look at the update

**Pull request merged**